app/keycloak.js

```diff
@@ -90,13 +93,57 @@ const onKeycloakEvent = (store) => (event, error) => {
 const onKeycloakTokens = (store) => (tokens) => {
   if (tokens?.token) {
     store.dispatch(sync.keycloakTokensReceived(tokens));
-    api.user.saveSession(keycloak?.tokenParsed?.sub, keycloak?.token, {
-      noRefresh: true,
-    });
-    if (!store.getState().loggedInUser) {
-      store.dispatch(async.doLogin());
+    let tokenParsed;
+    try {
+      tokenParsed = jose.decodeJwt(tokens?.token);
```

Comment:
Can there ever be more than one token here?

Reply:
There is a refresh token and ID token that are also on the `tokens` object, but we don't use them for explicitly for the session.